### PR TITLE
New Login/Welcome screen design

### PR DESCRIPTION
**Pull request by Cervera96**

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -36,6 +36,7 @@ target 'FriendlyPlans' do
   pod 'Firebase/Core', '~> 5.20.2'
   pod 'Firebase/Firestore', '~> 5.20.2'
   pod 'Firebase/Auth', '~> 5.20.2'
+  pod 'Firebase/Storage'
```
Comment (bklukaczewski) [inline ios/Podfile]: I'm surprised it wasn't connected for iOS. 
Can we lock the version like with the rest of Firebase libraries?

File: src/screens/resetPassword/ResetPasswordScreen.tsx [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
 
 export class ResetPasswordScreen extends React.PureComponent {
   render() {
     return (
-      <FullScreenTemplate padded narrow>
-        <StyledText style={styles.guideText}>{i18n.t('resetPassword:guide1')}</StyledText>
-        <StyledText style={styles.guideText}>{i18n.t('resetPassword:guide2')}</StyledText>
-        <ResetPasswordForm />
-      </FullScreenTemplate>
+      <Fragment>
```
Comment (bklukaczewski) [inline src/screens/resetPassword/ResetPasswordScreen.tsx]: You could also use `<>...</>`

File: src/screens/signUp/SignUpForm.tsx [hunk 3]
```diff
@@ -44,39 +80,43 @@ export class SignUpForm extends React.PureComponent<Props> {
...
   },
-  link: {
-    color: palette.primary,
+  inputContainer: { borderWidth: 0, marginBottom: 12 },
+  input: {
+    backgroundColor: 'white',
+    borderRadius: 12,
+    height: 44,
+  },
+  signUpbutton: {
```
Comment (bklukaczewski) [inline src/screens/signUp/SignUpForm.tsx]: typo `signUpButton` capital Button

File: src/screens/signUp/SignUpForm.tsx [hunk 4]
```diff
@@ -44,39 +80,43 @@ export class SignUpForm extends React.PureComponent<Props> {
...
+  },
+  signUpbuttonContainer: {
     marginTop: 8,
     marginBottom: 20,
   },
-  link: {
-    color: palette.primary,
+  inputContainer: { borderWidth: 0, marginBottom: 12 },
+  input: {
+    backgroundColor: 'white',
```
Comment (bklukaczewski) [inline src/screens/signUp/SignUpForm.tsx]: use colors from palette please :)

**Comment by Cervera96** [comment]
image

**Pull request merged**